lib/docker-sync/dependencies/fswatch.rb

```diff
@@ -0,0 +1,14 @@
+module DockerSync
+  module Dependencies
+    module Fswatch
+      def self.available?
+        return @available if defined? @available
```

Comment:
Yep, it should not be in `Docker::Driver` neither imho, but it's the only way to avoid false positive of people **genuinely** using Docker on an Alpine Linux host 😕